traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.Steering.tpl.html

```diff
@@ -61,452 +63,388 @@
+                    <div class="form-group" ng-class="{'has-error': hasError(routingConfig.geoLimitCountries), 'has-feedback': hasError(routingConfig.geoLimitCountries)}" ng-show="deliveryService.geoLimit === 1 || deliveryService.geoLimit === 2">
+                        <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="geoLimitCountries">Geo Limit Countries<div class="helptooltip">
+                            <div class="helptext">How (if at all) is this service to be limited by geography. Example Country Codes: <abbr title="Canada">CA</abbr>, <abbr title="India">IN</abbr>, <abbr title="Puerto Rico">PR</abbr>.</div>
+                        </div>
+                        </label>
+                        <div class="col-md-10 col-sm-10 col-xs-12">
+                            <input id="geoLimitCountries" name="geoLimitCountries" type="text" class="form-control" ng-model="deliveryService.geoLimitCountries" maxlength="255" pattern="[A-Z]+(,[A-Z]+)*">
+                            <small class="input-error" ng-show="hasPropertyError(routingConfig.geoLimitCountries, 'maxlength')">Too Long</small>
+                            <aside class="current-value" ng-if="settings.isRequest" ng-show="open() && deliveryService.geoLimitCountries != dsCurrent.geoLimitCountries">
+                                <h3>Current Value</h3>
+                                <pre>{{::dsCurrent.geoLimitCountries}}</pre>
+                            </aside>
+                        </div>
                     </div>
-                    </label>
-                    <div class="col-md-10 col-sm-10 col-xs-12">
-                        <input id="checkPath" name="checkPath" type="text" class="form-control" ng-model="deliveryService.checkPath" maxlength="255">
-                        <small class="input-error" ng-show="hasPropertyError(deliveryServiceForm.checkPath, 'maxlength')">Too Long</small>
-                        <span ng-show="hasError(deliveryServiceForm.checkPath)" class="form-control-feedback"><i class="fa fa-times"></i></span>
-                        <aside class="current-value" ng-if="settings.isRequest" ng-show="open() && deliveryService.checkPath != dsCurrent.checkPath">
-                            <h3>Current Value</h3>
-                            <pre>{{::dsCurrent.checkPath}}</pre>
-                        </aside>
+                    <div class="form-group" ng-class="{'has-error': hasError(routingConfig.geoLimitRedirectURL), 'has-feedback': hasError(routingConfig.geoLimitRedirectURL)}">
+                        <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="geoLimitRedirectURL">Geo Limit Redirect URL<div class="helptooltip">
+                            <div class="helptext">
+                                Traffic Router will redirect to this URL when Geo Limit check fails.
+        
```

Comment:
The readthedocs page will redirect this to HTTPS automatically - but it'd be nice to let users skip that step by taking them to the HTTPS page from the start.

Reply:
confused. is this not correct?

`See <a href="https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#tr-ngb" target="_blank">GeoLimit Failure Redirect</a> feature...
`

Reply:
The issue is with `http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#pattern-based-consistenthash` - but because you updated your PR (several times) while I was reviewing, GitHub became very confused about where certain comments are anchored.

Reply:
oh, i see. will fix

Reply:
i'll look for all instances of `http` on the ds forms